literature/wang_1997_lateral_1/wang_1997_lateral_1_p1_1a.yaml

```diff
@@ -0,0 +1,102 @@
+# This file describes a figure from a publication containing electrochemical data.
+# For the `echemdb/website` project, the source, curator and electrochemical system is mandatory.
+# Figure description is optional. It is created automatically by the cv module of svgdigitizer
+
+source:
+    version: 1
+    doi:  #only the number without preceeding doix or https:

```

Comment:
please add DOI

Reply:
The publication seems to have no DOI. See: https://www.osti.gov/biblio/537264

Reply:
Then I guess we should use in this case: www.osti.gov/biblio/537264

Reply:
```suggestion
    doi: www.osti.gov/biblio/537264 #only the number without preceeding doix or https:
```